.drone.yml

```diff
@@ -64,3 +64,24 @@ pipeline:
     when:
       branch: master
       event: push
+
+  deploy_production_cronjobs:
+    group: deploy
+    image: quay.io/mongodb/drone-helm:v2.14.1-0.1.0
+    # The release name should be unique across the namespace, the app or repo name is recommended
+    release: selected-tests
+    namespace: server-tig
+    environment:
+      - API_SERVER=https://api.prod.corp.mongodb.com
+    prefix: prod
+    secrets: [ prod_kubernetes_token ]
+    helm_repos: mongodb=https://10gen.github.io/helm-charts
+    chart: mongodb/cronjobs
+    chart_version: 1.3.0
```

Comment:
Can you update this to use the latest version of the cron job helm chart? It seems like the newest one is 1.5.1

https://github.com/10gen/helm-charts/blob/master/charts/cronjobs/CHANGELOG.md